CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Breaking changes
 
+- Improved on parity fix of `Mina.Localblockchain` and`Mina.Network` to make `.send()` and `.wait()` throw an error by default if the transaction was not successful. https://github.com/o1-labs/o1js/pull/1480
+  - Changed `Transaction.isSuccess` to `Transaction.status` to better represent the state of a transaction.
+  - `Transaction.safeSend()` and `PendingTransaction.safeWait()` are introduced to return a `IncludedTransaction` or `RejectedTransaction` object without throwing errors.
```

Comment:
since both of them are in the same release, it would be nice to collapse that changelog with the changes from #1422 mentioned below

for example not have both
* `.send()` no longer throws an error ...
* `.send()` throws an error by default ...

but just
* `.send()` when using `Mina.Network` is now consistent with `LocalBlockchain` in that it throws an error

Reply:
great point! done!